requirements/dev.pip

```diff
@@ -15,7 +15,7 @@ pytest>=2.3
 webtest;        python_version != '2.5'
 mock;           python_version <= '3.2'
 gevent>=1.1b4;  python_version >= '3'
-twine>=1.6.1
+twine>=1.7,<=1.7.4
```

Comment:
So, this shouldn't be registering as a change, and I don't think it would be with a proper rebase. This is from PR #158, which has already been merged. It was originally added because `twine` 1.8 breaks our `test_server` tests due to a new call signature.